unified-installer/astra-unified-installer.sh

```diff
@@ -197,6 +302,17 @@ set_log_level() {
     [ "$LOG_LEVEL" == "fatal" ] && LOG_LEVEL="$__FATAL"
 }
 
+ingest_config_string() {
+    local -r config_str="$1"
+    [ -z "$config_str" ] && fatal "no config string given"
+    local -r tmp_env="$__TMP_ENV"
+
+    echo "$config_str" > "$tmp_env"
+    # shellcheck disable=SC1090
+    source "$tmp_env"
+    rm -f "$tmp_env" &> /dev/null
+}
```

Comment:
This function and the changes below are where we reverse the priority of CLI vars VS config vars. It's commented pretty heavily since it makes use of weird-looking bash magic.